inputs/make_metadata/01_srp_id_to_runinfo.sh

```diff
@@ -0,0 +1,2 @@
+wget https://raw.githubusercontent.com/nf-core/fetchngs/c318ae12fcb972ff28f6d38edd3165c88b6d0b3f/bin/sra_ids_to_runinfo.py
```

Comment:
Is there a reason to download the script as part of the pipeline rather than adding the script to the repository? Their MIT license is pretty permissive, but I do think that you're at least supposed to say "hey this code is from here with this license"

Reply:
ya i always waffle between strategies...I chose to download here so that provenance was clear. Also I only use this script to generate metadata, and while I wanted to document how I made that metadata, the metadata table is packaged with the repository so this code shouldn't need to be rerun. When my pipeline relies more directly on a script, I usually download it, add it to my projects git repo, and and attribute it's origin.